docs/reference/glossary.md

```diff
@@ -32,6 +32,27 @@ An image that has no parent is a **base image**.
 [boot2docker](http://boot2docker.io/) is a lightweight Linux distribution made
 specifically to run Docker containers. The boot2docker management tool for Mac and Windows was deprecated and replaced by [`docker-machine`](#machine) which you can install with the Docker Toolbox.
 
+## bridge
+
+In terms of generic networking, a bridge is a Link Layer device which forwards
+traffic between networks. A bridge can be a hardware device or a software device
+running within a host machine's kernel.
+
+In terms of Docker, a bridge network uses a software bridge which allows
+containers connected to the same bridge network to communicate, while providing
+isolation from containers which are not connected to that bridge network.
+Containers which are not connected to the same bridge network can only
+communicate with each other if the host machine's network configuration and
+routing rules allow them to do so.
+
+The default bridge network, which is also named `bridge`, behaves differently
+from user-defined bridge networks. Containers connected to the default `bridge`
+network cannot communicate with each other across the bridge unless they are
```

Comment:
This is not true in these terms. The difference is that service discovery is not enabled on the default `bridge` network, compared to the other bridge driver user created networks.
Maybe we should specify  `cannot communicate with each other using their name` ?